File: db4oME/com/db4o/StaticField.java
/* Copyright (C) 2004   db4objects Inc.   http://www.db4o.com */

package com.db4o;

/**
 * @exclude
 * @persistent
 */
public class StaticField implements Internal4{
    public String name;
    public Object value;
    
    public StaticField(){
    }
    
    public StaticField(String name, Object value){
        this.name = name;
        this.value = value;
    }
}

File: db4o.net/Db4objects.Db4o/Db4objects.Db4o/StaticField.cs
/* Copyright (C) 2004 - 2011  Versant Inc.  http://www.db4o.com */

using Db4objects.Db4o;

namespace Db4objects.Db4o
{
	/// <exclude></exclude>
	/// <persistent></persistent>
	public class StaticField : IInternal4
	{
		public string name;

		public object value;

		public StaticField()
		{
		}

		public StaticField(string name_, object value_)
		{
			name = name_;
			value = value_;
		}
	}
}
